cache/lruCache.go

```diff
@@ -244,7 +244,7 @@ func (c *lruCache) SetWithExpiration(key interface{}, value interface{}, expirat
 	ent.value = value
 	ent.expiration = exp
 
-	atomic.AddUint64(&c.stats.Writes, 1)
+	c.stats.Writes++
```

Comment:
This was the only place that was using an atomic increment against `c.stats`. It wasn't clear why the atomic increment was necessary.

Reply:
This isn't the right fix. The atomic increment is necessary otherwise the counter will go out of whack. This would probably be flagged by Go's race detector as an error as well.

The right approach is to ensure proper alignment of the struct.

Reply:
Hmmm, seems like a leftover, probably from cloning the ttlCache.go code initially.

Reply:
Forgot to point out earlier that this is also within `c.Lock()`